### ISSUE TITLE
Add "notes" types with colours

### ISSUE THREAD

**Issue by bep** [gohugoio/hugoDocs#90]
![image](https://user-images.githubusercontent.com/394382/28619240-b504ecc4-7207-11e7-84a7-78e8b8556492.png)

The above is from the Jekyll docs. We have the "notes" shortcode, which is blue, but I kind of whish we had some more -- at least one "pro tip" in another colour. 

Oh... and those Jekyll background gradients looks nice ...

**Comment by rdwatters** [comment]
We already have `note`,`tip`, and `warning` admonitions (i.e., reStructuredText directives): 
https://gohugo.io/contribute/documentation/#admonitions

Source: https://github.com/gohugoio/hugoDocs/tree/master/layouts/shortcodes

So I think this is just a matter of visual styling at this point. What are your thoughts @budparr?

**Comment by jamesrhea** [comment]
Just an observation: currently, `note` and `warning` have the same visual appearance [in the docs](https://gohugo.io/contribute/documentation/#admonitions).

I guess that's a template issue since the underlying classes are different...?

**Comment by budparr** [comment]
I agree they need to be differentiated, ala the OP. I've been weighed down with some migrations (from an old SSG to Hugo) and hope to return to this soon, unless, of course, someone beats me to it.
